src/main/java/com/avengers/gamera/service/ArticleService.java

```diff
@@ -78,13 +78,12 @@ public ArticleGetDto createArticle(ArticlePostDto articlePostDto) {
         article.setAuthor(userService.findUser(articlePostDto.getAuthorId()));
         EArticleType articleType = articlePostDto.getType();
 
-        if (articleType == EArticleType.valueOf("REVIEW")) {
+        if (articleType == EArticleType.REVIEW) {
+            article.setGame(gameService.findActiveGame(articlePostDto.getGameId()));
```

Comment:
```suggestion
            article.setGame(Game.Builder().id(articlePostDto.getGameId()).build());
```

不需要查询一次,  直接放入id 会自动存的  article用了@manytoone